### PR TITLE
feature/blockquote

### PR DESCRIPTION
**Pull request by yunusfirat**

### CODE REVIEW

File: src/blockquote/Blockquote.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   */
+    className?: string;
+    /**
+   * content of the blockquotes
+   */
+    text: string | number;
+    /**
+   * footer of the blockquotes
+   */
+    footer: string | number;
```
Comment (daniele-zurico) [inline src/blockquote/Blockquote.tsx:17]: I think we can make this one optional

File: stories/BlockQuote/ClassBased.stories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Blockquote } from '../../src/blockquote/Blockquote';
+/**
+ * In this section we're using the blockquote component providing the **GovUk style** passing the relative `className.
+ * Feel free to use your own css to style the blockquote as you prefer.
+ */
```
Comment (daniele-zurico) [inline stories/BlockQuote/ClassBased.stories.js:5]: I don't think govUK has a style/class for blockquote.... you need to change this description. (in storybook there're other examples where the component are not govUK keep an eye on them.

File: stories/BlockQuote/ClassBased.stories.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      showPanel: true,
+    },
+  },
+  tags: ['autodocs'],
+};
+
+export const Basic = {
+  name: 'Basic',
+  args: {
+    className: 'blockquote-reverse',
```
Comment (daniele-zurico) [inline stories/BlockQuote/ClassBased.stories.js:20]: this class do not exists anywhere you need to implement it... probably better to call it only blockquote... not sure why reverse

File: stories/BlockQuote/ClassBased.stories.js (hunk 2 above)
Comment (yunusfirat) [reply stories/BlockQuote/ClassBased.stories.js]: I have make changes now and pushed it.

File: stories/BlockQuote/Documentation.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { Meta, Canvas, ArgTypes } from '@storybook/blocks';
+import * as BlockquoteStories from './UnStyled.stories';
+
+<Meta title="DCXLibrary/Typography/Blockquote/Documentation" />
+
+# Blockquote
+
+A blockquote component ready to use in your Typography
+Blockquote is UI/UX agnostic so you need to provide your style to have the look and feel you prefer.
```
Comment (daniele-zurico) [inline stories/BlockQuote/Documentation.mdx:9]: press space twice

File: stories/BlockQuote/ClassBased.stories.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      showPanel: true,
+    },
+  },
+  tags: ['autodocs'],
+};
+
+export const Basic = {
+  name: 'Basic',
+  args: {
+    className: 'blockquote',
```
Comment (daniele-zurico) [inline stories/BlockQuote/ClassBased.stories.js:19]: you need to create a new file called blockquoteStyle.css, define the style there and import it here:
```js
import './blockquoteStyle.css';
```

**Review by daniele-zurico — APPROVED** [review]

**Pull request merged**